test/tests.cpp

```diff
@@ -108,3 +108,8 @@ TEST(INIReader, read_big_file) {
         EXPECT_EQ(v, i);
     }
 }
+
+TEST(INIReader, dulicate_keys) {
+    EXPECT_THROW(INIReader r{"./fixtures/duplicate_keys.ini"},
```

Comment:
no need create a named instance, `INIReader{"./fixtures/duplicate_keys.ini"}` would be more straightforward.